### PR TITLE
Update annotations and rename CSV yaml file for certified OLM

### PR DESCRIPTION
**Pull request by lisyukanna**

### CODE REVIEW

File: deploy/olm-catalog/ibm-block-csi-operator/1.8.0/metadata/annotations.yaml [hunk 1]
```diff
@@ -5,3 +5,5 @@ annotations:
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: ibm-block-csi-operator
+  com.redhat.openshift.versions: "v4.7-v4.9"
+  com.redhat.delivery.operator.bundle: true
```
Comment (oriyarde) [inline deploy/olm-catalog/ibm-block-csi-operator/1.8.0/metadata/annotations.yaml:9]: looks like a newline is missing
image

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Review by oriyarde — APPROVED** [review]
@barakda it seems we [don't need](https://github.com/redhat-openshift-ecosystem/certified-operators#directory-structure) the [Dockerfile](https://github.com/IBM/ibm-block-csi-operator/blob/9baf41e85f1e4684ea776ef488b7ecd1eebfd1f7/deploy/olm-catalog/ibm-block-csi-operator/1.8.0/bundle-1.8.0.Dockerfile) either - please open a ticket for 1.9.0 to remove it. and consider removing all the bundles before 1.8.0 as well

File: deploy/olm-catalog/ibm-block-csi-operator/1.8.0/metadata/annotations.yaml (hunk 1 above)
Comment (barakda) [reply deploy/olm-catalog/ibm-block-csi-operator/1.8.0/metadata/annotations.yaml]: @lisyukanna , apparently few more changes are required here:
1. remove the quote marks from openshift version part "v4.7-v4.9"
2. remove the last line, it is not needed

**Comment by barakda** [comment]
@lisyukanna ,also we need to replace the version tag with SHA in the csv file so replace 1.8.0 tag with sha256:df209e133fb5f25aa6d28afe12537a8e963a1ab9fbb2452d4314913ca9a5dde5 in all relevant places

**Pull request merged**